src/index_notation/index_notation.cpp

```diff
@@ -2339,6 +2371,67 @@ bool isReductionNotation(IndexStmt stmt, std::string* reason) {
   return isReduction;
 }
 
+bool isReductionNotationScheduled(IndexStmt stmt, ProvenanceGraph provGraph, std::string* reason) {
+  INIT_REASON(reason);
+
+  if (!isa<Assignment>(stmt)) {
+    *reason = "reduction notation statements must be assignments";
+    return false;
+  }
+
+  if (!isValid(to<Assignment>(stmt), reason)) {
+    return false;
+  }
+
+  // Reduction notation until proved otherwise
+  bool isReduction = true;
+
+  util::ScopedMap<IndexVar,int> boundVars;  // (int) value not used
```

Comment:
If the int value is not used, could this be replaced with a ScopedSet then?

Reply:
This is the same code from `isReductionNotation(stmt)` copied. I will change it in both places.